src/test-fbp/converter-error-expected.txt

```diff
@@ -0,0 +1 @@
+WRN: ./src/lib/common/sol-types.c:242 sol_irange_division() Division by zero: 10, 0
```

Comment:
@cmarcelo I'm not sure about that. We may miss some unexpected warnings this way. In this case I really wanted to force an error to create an error packet so nodes that handle error packets could be tested. But usually, if a warning shows up while we're running these tests, it means something gone wrong. It's a test suite after all. We're using it to test node types code and cover our codebase. I'm more compelled to work with a solution like proposed by @lpereira to make sure we're not masking any problems.